benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml

```diff
@@ -20,7 +42,7 @@ spec:
     name: hammerdb
     args:
       pin: {{ pin }} # enable for nodeSelector
-      pin_node: "{{ pin_node1 }}"
+      pin_node: "{{ pin_node2 }}"
```

Comment:
Why this change?

Reply:
We have the LSO disk available only on node2.

Reply:
Then change pin_node1 in the environment.  The repository shouldn't contain configuration-specific information.

Reply:
We have only 1 disk for LSO and its located on in one of the nodes for cost saving, that the reason we need to pin specific node.

Reply:
Right.  Change the value of pin_node1 in the Github action, not in the code.

Reply:
I prefer to keep it as is, the change is only for the hammerdb vm workload